Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -593,8 +595,8 @@ bool Stringifier::Holder::appendNextProperty(Stringifier& stringifier, StringBui
             builder.append(',');
         stringifier.startNewLine(builder);
 
-        // Append the property name.
-        builder.appendQuotedJSONString(propertyName.string());
+        // Append the property name, colon, and space.
+        builder.appendQuotedJSONString(*propertyName.uid());
```

Comment:
Just realized I can speed this up a bit more by changing `appendQuotedJSONString` to take a `StringView`.